sunpy/image/transform.py

```diff
@@ -325,8 +346,58 @@ def _rotation_skimage(image, matrix, shift, order, missing, clip):
     return rotated_image
 
 
+@add_rotation_function("cv2", allowed_orders={0, 1, 3},
```

Comment:
Why `'cv2'` and not `'opencv'`?

Reply:
\<shrug\>  Extrapolating from the single point of `'skimage'` instead of `'scikit-image'`, it seems that we are preferring the import name.

Reply:
I also like "cv2" over "opencv".

Reply:
I really don't like "cv2" it's completely impenetrable to people who haven't used opencv before, unlike opencv it doesn't give you sensible things when you search for it (unlike skimage which is the first result). While it might be the module name, I would argue that whoever decided that you should do `import cv2` was also not of sound mind.

Reply:
If I recall correctly, the reason that the package is called `cv2` is because they changed the API from their original `cv` package.  It has nothing to do with the version of OpenCV.

If we want, we could actually support both `'cv2'` and `'opencv'`.  We could also support both `'skimage'` and `'scikit-image'`.  Why not!